backend/app/infrastructure/routes/user.py

```diff
@@ -192,12 +196,14 @@ class AntisybilStatus(OctantResource):
     @ns.response(200, "User's cached antisybil status retrieved")
     def get(self, user_address: str):
         app.logger.debug(f"Getting user {user_address} cached antisybil status")
+
         antisybil_status = get_user_antisybil_status(user_address)
         app.logger.debug(f"User {user_address} antisybil status: {antisybil_status}")
         if antisybil_status is None:
             return {"status": "Unknown"}, 404
-        score, expires_at = antisybil_status
+        (score, expires_at), (has_sbt, _) = antisybil_status
```

Comment:
when the controller returns None, it'll raise the error here probably; but firstly consider it: https://github.com/golemfoundation/octant/pull/393/files#r1732355777, maybe it'll ease this pain in the ass

Reply:
+1